person.rb

```diff
@@ -1,3 +1,21 @@
 class Person
+  attr_reader :id
+  attr_accessor :name, :age
 
-end
\ No newline at end of file
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? or @parent_permission
+  end
+
+  private
+
+  def of_age?
```

Comment:
- [ ] I think you accidentally misplaced the name of this method, this should be `is_of_age?` which you have referenced in your public method too, please fix this :+1: